src/main/java/com/liferay/faces/test/selenium/applicant/ApplicantTesterBase.java

```diff
@@ -179,6 +184,7 @@ public void runApplicantPortletTest_C_FirstNameField() {
 		WaitingAsserter waitingAsserter = getWaitingAsserter();
 		waitingAsserter.assertElementNotDisplayed(firstNameFieldErrorXpath);
 		browserDriver.clearElement(firstNameFieldXpath);
+		lastNameFieldClick = browserDriver.createClickElementAction(lastNameFieldXpath);
```

Comment:
@ngriffin7a, sorry, I put that in cause I was getting errors when ButterFaces used the blur event. Since we removed that code, I removed the line you mentioned here: https://github.com/ngriffin7a/liferay-faces-test-selenium/pull/38